tests/layers_views/layers/MultiscaleImageLayerBase.spec.js

```diff
@@ -27,19 +32,78 @@ test('MultiscaleImageLayerBase', t => {
       colorValues: [[0, 1, 1]],
       channelIsOn: [true],
       tileSize: 2,
-      imageWidth: 4,
-      imageHeight: 4,
+      loader: [
+        {
+          getTileData,
+          dtype: 'Uint32',
+          shape: [1, 1, 1, 2, 2]
+        }
+      ],
       dtype: 'Uint32',
+      getTileData
+    },
+    onBeforeUpdate: ({ testCase }) => t.comment(testCase.title),
+    onAfterUpdate: ({ subLayers }) =>
+      t.ok(
+        subLayers.length > 0
+          ? subLayers.every(layer => layer.constructor.name === 'XRLayer')
+          : true
+      )
+  });
+  await testLayerAsync({
+    Layer: MultiscaleImageLayerBase,
+    testCases,
+    onError: t.notOk,
+    viewport: view.makeViewport({
+      height: 4,
+      width: 4,
+      viewState: { target: [2, 2, 0], zoom: 0, width: 4, height: 4 }
+    })
+  });
+  t.end();
+});
+
+test('MultiscaleImageLayerBaseBitmapLayer', async t => {
+  const view = new OrthographicView({
+    id: 'ortho',
+    controller: true,
+    height: 4,
+    width: 4,
+    target: [2, 2, 0],
+    zoom: 0
+  });
+  const getTileData = async () => ({
+    data: new Uint8Array([0, 0, 0, 2, 2, 2, 1, 1, 1, 2, 2, 2]),
+    width: 2,
+    height: 2
+  });
+  const testCases = generateLayerTests({
+    Layer: MultiscaleImageLayerBase,
+    assert: t.ok,
+    sampleProps: {
       // eslint-disable-next-line no-unused-vars
-      getTileData: () =>
-        new Promise(() => [new Uint32Array([0, 1, 2, 3])])
+      getTileData,
+      loader: [
+        {
+          getTileData,
+          dtype: 'Uint8',
+          shape: [1, 1, 1, 2, 2, 4],
+          meta: { photometricInterpretation: 0 }
+        }
+      ]
     },
-    onBeforeUpdate: ({ testCase }) => t.comment(testCase.title)
+    onBeforeUpdate: ({ testCase }) => t.comment(testCase.title),
+    onAfterUpdate: ({ subLayers }) =>
+      t.ok(
+        subLayers.length > 0
+          ? subLayers.every(layer => layer.constructor.name === 'BitmapLayer')
+          : true
+      )
   });
-  testLayer({
+  await testLayerAsync({
```

Comment:
Needed to use `async` to be sure that our tests actually rendered their sublayers.